ui/common/Input.tsx

```diff
@@ -100,15 +100,15 @@ export default function Input({
         onChange={handleChange}
         value={textAreaValue}
         spellCheck="false"
-        className={`absolute top-0 left-0 h-full w-full resize-none overflow-hidden bg-transparent font-space-mono text-[18px] leading-[180%] tracking-[1px] text-transparent outline-none md:text-[30px] md:tracking-[5px]`}
+        className={`absolute top-0 left-0 h-full w-full max-w-[95vw] resize-none overflow-hidden bg-transparent font-space-mono text-[18px] leading-[180%] tracking-[1px] text-transparent outline-none md:text-[30px] md:tracking-[5px]`}
```

Comment:
Hey @benalleng, any particular reason why you've chosen `95vw` here?

Reply:
So at first, I set the max width to the screen width but I noticed that the input wasn't respecting any padding from its parent elements and was filling the screen completely. Now that you've pointed it out I realize that may mean there is an overflow error I just now tried `word-break: break-all` and that also fixes the problem. In addition to the input now respecting CSS padding from its parent elements. Thanks for pushing me to check twice on this one!